roles/openshift_metrics/tasks/install_metrics.yaml

```diff
@@ -11,19 +12,23 @@
   loop_control:
     loop_var: include_file
   when: "not {{ openshift_metrics_heapster_standalone | bool }}"
+  run_once: True
 
 - name: Install Heapster Standalone
   include: install_heapster.yaml
   when: "{{ openshift_metrics_heapster_standalone | bool }}"
+  run_once: True
```

Comment:
I think reading through install_heapster.yaml and generate_heapster_certificates.yaml they should all be run_once: true so we should remove it within install_heapster.yaml.

@jcantrill can you confirm?